### DISCUSSION TITLE
Is it possible to add keybindings to the quick create?

### DISCUSSION THREAD

**Discussion by johntrickett86** [awcodes/filament-quick-create#60] [Ideas]
In the same way that we can attach global key bindings to the search, it would be really cool if we could do the same with the quick create. Looking at the filament docs, it seems keybindings can be added to actions - but I'm unsure on the best way to achieve this with the plugin. Any ideas?

**Comment by awcodes** [comment]
I'm not sure how that would work, you'd have to have a different key binding assigned to each individual CreateAction for each of your resources, otherwise the binding would trigger them all.

**Reply by johntrickett86** [reply]
@awcodes I was thinking more of a binding to open the dropdown - you can use keyboard navigation from there.

**Reply by awcodes** [reply]
Let me think about it. Not sure what a good default would be. Definitely don't want to use 'Q' 🤣

I'm certainly open to PRs if you can get it working as an opt in feature.

**Reply by johntrickett86** [reply]
@awcodes Let me have a think about it. I'll try a few things out and if I find a suitable solution I'll create a PR 👍

**Comment by johntrickett86** [comment]
@awcodes I figured it out. Submitted a PR: https://github.com/awcodes/filament-quick-create/pull/61
